src/components/common/MoodIcon.jsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image";
+
+const MoodIcon = ({src, alt, width, height}) => (
+        <Image
+            src={src}
+            alt={alt}
+            width={width}
+            height={height}
+        />
+);
+
+const MoodIconContainer = ({mood, width, height}) => {
```

Comment:
It's better to have a default value for parameters { mood = "", widrth = 100, height = 100 }

Reply:
Thank you for sharing this!
I set the default value to the parameters instead.